core/core.go

```diff
@@ -501,25 +497,29 @@ func (c *Core) UpdateEnvironment(namespace string, organization string, elements
 }
 
 func (c *Core) GetNamespaceDatabaseConfiguration(namespace string) models.Database {
-	databaseConf := c.Conf.VampConfiguration.Persistence.Database
 
 	return models.Database{
 		Sql: models.SqlConfiguration{
-			Database:          Namespaced(namespace, databaseConf.Sql.Database),
-			Table:             Namespaced(namespace, databaseConf.Sql.Table),
-			User:              Namespaced(namespace, databaseConf.Sql.User),
-			Password:          Namespaced(namespace, databaseConf.Sql.Password),
-			Url:               Namespaced(namespace, databaseConf.Sql.Url),
-			DatabaseServerUrl: Namespaced(namespace, databaseConf.Sql.DatabaseServerUrl),
+			Database: Namespaced(namespace, c.Conf.DatabaseName),
+			Table:    Namespaced(namespace, c.Conf.DatabaseTable),
+			User:     c.Conf.DatabaseUser,
+			Password: c.Conf.DatabasePassword,
+			Url:      Namespaced(namespace, c.Conf.DatabaseURL),
 		},
-		Type: databaseConf.Type,
+		Type: c.Conf.DatabaseType,
 	}
 }
 
 func (c *Core) GetNamespaceKeyValueStoreConfiguration(namespace string) *models.KeyValueStoreConfiguration {
-	model := c.Conf.VampConfiguration.Persistence.KeyValueStore
-	model.BasePath = Namespaced(namespace, c.Conf.VampConfiguration.Persistence.KeyValueStore.BasePath)
-	return &model
+
+	return &models.KeyValueStoreConfiguration{
+		Type:     c.Conf.KeyValueStoreType,
+		BasePath: Namespaced(namespace, c.Conf.KeyValueStoreBasePath),
+		Vault: models.VaultKeyValueStoreConfiguration{
```

Comment:
Can you revert this back?
If you do that you are deleting other parameters like certificates and kv modes.